crates/runner/src/lib.rs

```diff
@@ -251,6 +251,12 @@ impl StoreContext {
     }
 }
 
+pub enum UseExportedFn {
+    EvalBytecode,
+    Invoke,
+    InvokeWithFn(&'static str),
```

Comment:
Just a minor comment here - could this be consolidated into:

```rust
pub enum UseExportedFn {
  Invoke(Option<&'static str>)
  ...
}
```

Reply:
Yes, I can make that change.